### PR TITLE
Mkerns/caseflow 2066 add visn orgs

### PR DESCRIPTION
**Pull request by mkerns355**
Resolves [CASEFLOW-2066](https://vajira.max.gov/browse/CASEFLOW-2066)

### Description
Added VISN orgs to the seeds file.

### Acceptance Criteria
- [ ] Code compiles correctly

### Screenshot
image

### CODE REVIEW

File: db/seeds/veterans_health_administration.rb [hunk 1]
```diff
@@ -30,5 +32,108 @@ def setup_program_offices!
...
+          url: "VA Midwest Health Care Network",
+          type: "VhaRegionalOffice"
+        }
+      ]
+    end
+
+    def create_visn_org_teams
+      visn_orgs.each do |org|
+        org[:url]&.parameterize&.dasherize
+        VhaRegionalOffice.create!(org)
```
Comment (leikkisa) [inline db/seeds/veterans_health_administration.rb:134]: I think it'd be useful to add users to these (one regular and one admin is good although one regular user is enough!). In the real world one user wouldn't likely be a member of multiple ROs but for the sake of seeds, adding the same user is probably fine.

**Review by leikkisa — APPROVED** [review]
One non-blocking comment (we can always add users later), otherwise LGTM!

File: db/schema.rb [hunk 2]
```diff
@@ -1534,8 +1534,6 @@
     t.datetime "submitted_at"
     t.datetime "updated_at", null: false
     t.datetime "uploaded_to_vbms_at"
-    # To-do: consider removing this index, which is superceded by the
-    # addition of index_vbms_uploaded_documents_on_appeal_type_and_appeal_id
```
Comment (bgantick) [inline db/schema.rb:1538]: Should maybe leave this comment in. @wdrougas-tista I think this gets removed anytime anyone runs a migration

File: db/seeds/veterans_health_administration.rb [hunk 3]
```diff
@@ -30,5 +32,108 @@ def setup_program_offices!
         OrganizationsUser.make_user_admin(admin_user, org)
       end
     end
+
+    def visn_orgs
```
Comment (bgantick) [inline db/seeds/veterans_health_administration.rb:36]: Could maybe move these to a separate config file (negating the need for the rubocop disable)

File: db/schema.rb (hunk 2 above)
Comment (wdrougas-tista) [reply db/schema.rb:1538]: Good to know. If it does after merging this I will make PR to fix

**Pull request merged**